src/utilities.rs

```diff
@@ -33,7 +60,13 @@ impl Candidate {
     }
 }
 
-#[derive(Debug, Clone, Copy, Default, PartialEq, PartialOrd, Eq, Ord)]
+impl Distribution<Candidate> for Standard {
```

Comment:
I'm not sure where we are using them.
It seems we only generating random Node for now.

Would it be possible/sensible, to avoid decorating all these struct with this trait, and have a function generating random Nodes for example. I see here we have an hard-coded `Age(rng.gen_range(5, 101))` which probably do not make sense for all random tests we will want.

Is there another pattern that would work?

Reply:
The `Candidate` one is unused right now, so yes, I'm happy to remove that.

The others all used in the test, with the exception of `Age` which is only used inside the implementation of `Attributes`.

I'm not strongly in favour of implementing `Distribution` for all these other types, but it does seem the most sensible approach to me.

I take your point about `Age` - I was unsure about that too.  Do we have a decision for the actual Routing code as to what that should be?  Last I remember, we spoke of having it a `u8` with a fixed lower limit.